src/index.js

```diff
@@ -210,6 +213,28 @@ async function Main() {
             });
         });
     }
+
+    async function checkForUpdate () {
+        spinner.start("Checking for an Update...");
+        // Using Github API (https://docs.github.com/en/rest/reference/repos#releases)
+        // to get the releases data
+        const url = "https://api.github.com/repos/vasani-arpit/WBOT/releases";
+        const response = await fetch(url);
+
+        // Storing data in form of JSON
+        var data = await response.json();
+        var latestVersion = data[0].tag_name;
+        var latestVersionLink = `https://github.com/vasani-arpit/WBOT/releases/tag/${latestVersion}`;
+        var myVersion = 'v' + require('../package.json').version;
+        
+        spinner.stop("Checking for an Update... Completed");
+
+        if(latestVersion !== myVersion) {
```

Comment:
Let's say user is ahead of the `latestVersion`, in that case this condition will be true and user will see the message which is a wrong behavior.